index.js

```diff
@@ -169,17 +169,25 @@ module.exports = function(app) {
         var hours = ('00' + datetime.getHours()).slice(-2);
         var minutes = ('00' + datetime.getMinutes()).slice(-2);
         var seconds = ('00' + datetime.getSeconds()).slice(-2);
+
+        var day = ('00' +datetime.getUTCDate()).slice(-2);
+        var month = ('00' +(datetime.getUTCMonth() + 1)).slice(-2); //months from 1-12
+        var year = ('00' +datetime.getUTCFullYear()).slice(-2);
+
         return toSentence([
-          '$SKRMC', hours + minutes + seconds + '.020',
```

Comment:
I'm not sure about taking this out. It must have been there for a reason. Anyone know why? 

I also noticed that the decoder @ http://freenmea.net/decoder has the decimal point when you click "Get Random NMEA"

Reply:
Yes, it is a decimal point of seconds. The given timestamp format does not have miliseconds.